packages/example-getting-started/docs/4-todo-model.md

```diff
@@ -0,0 +1,104 @@
+### Building the Todo model
+
+The Todo model will be the object we use both as a Data Transfer Object (DTO) on
+the controller, and as a LoopBack model for the Legacy Juggler implementation.
+
+Create another folder in `src` called `repositories` and inside of that folder,
+create two files:
+- `index.ts`
+- `todo.repository.ts`
+
+>**NOTE:**
+The `index.ts` file is an export helper file; this pattern is a huge time-saver
+as the number of models in your project grows, because it allows you to point
+to the _directory_ when attempting to import types from a file within the target
+folder. We will use this concept throughout the tutorial!
+```ts
+// in src/models/index.ts
+export * from './foo.model';
+export * from './bar.model';
+export * from './baz.model';
+
+// elsewhere...
+
+// with index.ts
+import {Foo, Bar, Baz} from './models';
```

Comment:
Isn't there the danger of creating circular dependencies here? The examples use models, but what if they were something with complicated dependencies? I remember having to go and change the imports around so that they were imported straight from their original files instead of from `index.ts` to solve some problems I was having.